packages/theme/src/colors/common.ts

```diff
@@ -14,17 +14,30 @@ export const gray = {
   1000: '#000000',
 }
 
+export const neutral = {
```

Comment:
Should the changes in this file be included in the PR? I believe that updating the colors should be part of https://github.com/consolelabs/mochi-ui/pull/634